cfg/question_model.cfg

```diff
@@ -119,7 +119,8 @@ pddl_dict_question_objectsActions_file = ${data_path}/valid_questions_objectsAct
 pddl_connection_file = ${data_path}/empty.txt
 
 # ATTENTION: this is written every time by the IR and has to be reset everytime
-text_connection_file = ${data_path}/qa.text_features
+text_connection_file = ${data_path}/valid_predicates.text_features
+ir:text_connection_file = ${data_path}/qa.text_features
```

Comment:
I use `text_connection_file` as the place where to load the features from
`ir:text_connection_file` where to put answers' connections